cirq-core/cirq/circuits/circuit.py

```diff
@@ -1169,6 +1175,10 @@ def to_text_diagram_drawer(
             The TextDiagramDrawer instance.
         """
         qubits = ops.QubitOrder.as_qubit_order(qubit_order).order_for(self.all_qubits())
+        draw_cregs = bool(protocols.control_keys(self))
```

Comment:
I think this is from when we had external control keys. This should always be empty since that was dropped. Consider removing the code.

Reply:
No, this is purely for populating the qubit_map.